packages/at_secondary_server/lib/src/verb/handler/otp_verb_handler.dart

```diff
@@ -29,6 +29,10 @@ class OtpVerbHandler extends AbstractVerbHandler {
       HashMap<String, String?> verbParams,
       InboundConnection atConnection) async {
     final operation = verbParams['operation'];
+    if (verbParams[AtConstants.ttl] != null &&
+        verbParams[AtConstants.ttl]!.isNotEmpty) {
+      otpExpiryInMills = int.parse(verbParams[AtConstants.ttl]!);
+    }
     switch (operation) {
       case 'get':
```

Comment:
Should have a `default` block which will throw an exception for any operation other than `get`. Even though currently we only have one operation, switch statements should always cover all cases (which in this case means we need a default handler.)